src/main/java/com/hazelcast/security/SimpleCredentialsFactory.java

```diff
@@ -0,0 +1,16 @@
+package com.hazelcast.security;
+
+import com.hazelcast.nio.serialization.DataSerializableFactory;
+import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
+
+public class SimpleCredentialsFactory implements DataSerializableFactory {
+    static final int FACTORY_ID = 1;
```

Comment:
Would this factory ID conflict with the serializer factories in tests?

Reply:
No, it won't be effective at all unless you explicitly register it in the XML configuration of the member you are starting